troubleshooting/faq.md

```diff
@@ -162,28 +162,28 @@ So it’s important to understand the different storage driver options available
 
 We recommend that you use the `overlay` driver, rather than `devicemapper`; for help configuring this [please see the following tutorial](https://docs.docker.com/engine/userguide/storagedriver/overlayfs-driver/#configure-docker-with-the-overlayoverlay2-storage-driver)
 
-## Why to use npm Enterprise?
+## Why use npm Enterprise?
 
-1. npm Enterprise is single tenant vs. multi tenant -- which is important for some compliance heavy companies; npm Enterprise also allows you to have any number of scopes, and provides its own website, which can be useful for large organizations.
+1. npm Enterprise is single tenant vs. multi tenant -- which is important for companies with advanced compliance needs. npm Enterprise also allows you to have any number of scopes, and provides its own website, which can be useful for large organizations.
 
 2. npm Enterprise allows you to run npm’s infrastructure behind your company’s firewall.
 
-4. Security is a serious concern, and with npm Enterprise we have mitigated the risks by partnering with *Node Security Platform* to provide the security features many enterprise customers have been looking for.
+3. Integration with the *Node Security Platform* provides package-level analysis to assist enterprise customers with security risk mitigation.
 
-   You can read our [blog](http://blog.npmjs.org/post/146943134240/npm-add-ons) about it, and another article straight from the [Node Security Platform](https://medium.com/node-security/announcing-npm-enterprise-security-add-on-6dde303efb9f).
+   Read more about it on our [blog](http://blog.npmjs.org/post/146943134240/npm-add-ons) and this [Node Security Platform article.](https://medium.com/node-security/announcing-npm-enterprise-security-add-on-6dde303efb9f).
 
-5. npm Enterprise can significantly improve the efficiency of the private code that you write at your organization making it easier to share documentation and code, distribute code to your build servers, and to breakup your monolithic code-bases into easier-to-reason-about modules.
+4. Significantly improve the efficiency of your development process making it easier to share documentation and code, streamline your build process, and to breakup your monolithic code-bases into individual packages that are easier to maintain.
```

Comment:
@ritsingh - sorry I missed one change on the initial review, please remove "to" from "and to breakup your monolithic code-bases".